docker-compose.yml

```diff
@@ -91,6 +92,7 @@ services:
     - NET_ADMIN
     - AUDIT_WRITE
     - SYS_CHROOT
+    #- SYS_RAWIO # Only needed if you are using real FIDO2 tokens
```

Comment:
This should be in different commit and perhaps use "only needed if you require access to real devices like FIDO2 tokens"?